src/zeta/aca_oam_server.cpp

```diff
@@ -68,38 +68,54 @@ void ACA_Oam_Server::_deinit_oam_ofp()
 ACA_Oam_Server &ACA_Oam_Server::get_instance(){
   static ACA_Oam_Server instance;
   return instance;
-};
+}
+
+int ACA_Oam_Server::_validate_oam_message(oam_message *oammsg)
```

Comment:
looks like this function can simply return true for valid oam message and false otherwise.